Profiles - ZIB 2017/zib-VaccinationRecommendation-OrderStatus.xml

```diff
@@ -41,8 +41,13 @@
         <strength value="preferred" />
         <description value="OrderStatusCodelist" />
         <valueSetReference>
-          <reference value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.16.1.2--20171231000000" />

```

Comment:
Hoi Eduard, waarom is dat hier aangepast? Ik kan er niks over terugvinden in het ticket.